src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -1717,20 +1717,20 @@ def _validate_input(self, row: pd.Series):
             return (False, (f"Invalid parameter combination: {e}"))
 
     def _validate_single_bead_input(
-        self, machine: AdditiveMachine, material: AdditiveMaterial, row: pd.Series
+        self, machine: AdditiveMachine, material: AdditiveMaterial, input: pd.Series
     ) -> tuple[bool, str]:
-        """Test a single bead input from a row of a CSV file for valid input parameters.
+        """Validate that a single bead input can be created.
```

Comment:
The wording here is just a little bit awkward. 
```suggestion
        """Validate single bead input values.
```